srcpkgs/gst-libav/template

```diff
@@ -12,18 +12,10 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.0-or-later"
 homepage="https://gstreamer.freedesktop.org"
 distfiles="${homepage}/src/${pkgname}/${pkgname}-${version}.tar.xz"
-checksum=c724f612700c15a933c7356fbeabb0bb9571fb5538f8b1b54d4d2d94188deef2
+checksum=39a717bc2613efbbba19df3cf5cacff0987471fc8281ba2c5dcdeaded79c2ed8
 
 case "$XBPS_TARGET_MACHINE" in
 	*-musl) # Required by musl for M_SQRT1_2
 		CFLAGS="-D_XOPEN_SOURCE"
 		;;
 esac
-
-post_configure() {
-	case "$XBPS_TARGET_MACHINE" in
-		x86_64-musl) # compiling with -O3 triggers a gcc bug
```

Comment:
is the gcc bug fixed? or did they change their compile flags?

Reply:
Added it in meson.build just in case it does still exist, as the file that is being sed'd doesn't exist now.

Reply:
the `-O2` was for `x86_64-musl` only, you should test if it builds without the patch on that target and if yes, then there's no need for a patch

Reply:
It built fine w/o -O2 on x86_64-musl, so will remove patch then

Reply:
It would be nice to test it at runtime as well; the bug might not have been compile time.

Reply:
that's highly unlikely, this wording implies some kind of ICE or similar condition